test/jellyfish/room_test.exs

```diff
@@ -0,0 +1,55 @@
+defmodule Jellyfish.RoomTest do
+  use ExUnit.Case, async: true
+
+  alias Jellyfish.{Peer, Room}
+
+  @purge_timeout_s 1
```

Comment:
The way those tests are written right now will cause us to wait a couple of seconds until they are executed. I am wondering if we could make this better. We can discuss this offline